cmd/send-email/main.go

```diff
@@ -25,7 +28,18 @@ func main() {
         log.Fatal("Could not load config options.", err);
     }
 
-    err = email.Send(args.To, args.Subject, args.Body, false);
+    // If database cannot get course and using role based addressing
+    // Then, resolve users will return an error
+    // So, we do not error check on the get course method
+    course, err := db.GetCourse(args.Course);
+    emailTo, err := db.ResolveUsers(course, args.To);
+    if (err != nil) {
+        log.Fatal("Failed to resolve users:'%s', '%w'.", course.GetName(), err);
```

Comment:
Missing a space after ':'.

But that is actually moot. since courses should just be passed to a logging statement and not by name.
The logging infrastructure specifically recognizes courses and does extra work with them.
E.g., https://github.com/edulinq/autograder-server/blob/main/api/user/add.go#L81

Reply:
Unresolved.

This issue still exists.